lib/BouncyCheckboxGroup.tsx

```diff
@@ -12,18 +12,20 @@ import useStateWithCallback from "./helpers/useStateWithCallback";
 type CustomStyleProp = StyleProp<ViewStyle> | Array<StyleProp<ViewStyle>>;
 
 export interface ICheckboxButton extends IBouncyCheckboxProps {
-  id: number;
+  id: string | number;
 }
 
 interface IBouncyCheckboxGroupProps {
   style?: CustomStyleProp;
   initial?: number;
   data: ICheckboxButton[];
   onChange: (selectedItem: ICheckboxButton) => void;
+  checkboxProps?: IBouncyCheckboxProps;
```

Comment:
Hi Wrath!

I was using my own commit on my projects but it's better if I use this lib without my custom changes, if we can have my those 2 little changes.

Yes, we can use {...item}, but with checkboxProps, we can have props that are passed to all the checkboxes, without defining it every time for each checkbox.

In the [`example` project](https://github.com/WrathChaos/react-native-bouncy-checkbox-group/blob/master/example/App.tsx), we have 5 data items with

```jsx
{
    id: 0,
    text: "Watermelon",
    fillColor: "#ff7473",
    unfillColor: "#fbbfbb",
    iconStyle: _iconStyle("#fbbfbb"),
    textStyle: styles.textStyle,
    style: styles.verticalStyle,
    iconImageStyle: styles.iconImageStyle,
  },
```

The `textStyle`, `style` and `iconImageStyle` props have the same values for the 5 items. Instead of repeating them, we can only

```jsx
{
    id: 0,
    text: "Watermelon",
    fillColor: "#ff7473",
    unfillColor: "#fbbfbb",
    iconStyle: _iconStyle("#fbbfbb"),
} // data only with the specific styles
    
<BouncyCheckboxGroup
  data={staticData}
  onChange={(selectedItem: ICheckboxButton) => {
    console.log("SelectedItem: ", JSON.stringify(selectedItem));
  }}
  checkboxProps={{
    textStyle: styles.textStyle,
    style: styles.verticalStyle,
    iconImageStyle: styles.iconImageStyle,
  }}
/>
```

This really simplifies the code and allows data to only have real data, if the dev doesn't want to customize any individual checkbox. This property is optional, if the dev doesn't want to define a default for them, he can still apply the props to each data item.

Reply:
Let me check this evening after work :)